admin/load.php

```diff
@@ -311,9 +332,11 @@ function perflab_get_module_data( $module_file ) {
 	$module_dir = $matches[1];
 
 	$default_headers = array(
-		'name'         => 'Module Name',
-		'description'  => 'Description',
-		'experimental' => 'Experimental',
+		'name'            => 'Module Name',
+		'description'     => 'Description',
+		'experimental'    => 'Experimental',
+		'enabled-notice'  => 'Enabled Notice',
+		'disabled-notice' => 'Disabled Notice',
```

Comment:
Same comment as before about the message for why the module cannot be loaded. We probably shouldn't add new APIs as part of this PR, especially since I'm not sure this is the way we want to go for providing a customized message. Can you revert those two fields and instead, similarly to how you do it for custom message around loading the module, hard-code these messages into `admin/load.php` for this module specifically?

Reply:
Done 👍